.github/workflows/helm.yaml

```
@@ -0,0 +1,56 @@
+name: helm
+
+on:
+  pull_request:
+    branches: [ '*' ]
+    paths:
+      - 'charts/**'
+  push:
+    tags:
+      - 'registry-operator-v*'
+
+env:
+  REGISTRY: ghcr.io
+  REPOSITORY: registry-operator
+
+permissions:
+  contents: write
+  packages: write
+
+jobs:
+  linters:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - uses: stackrox/kube-linter-action@v1
```

Consider using the newest version
```suggestion
      - uses: stackrox/kube-linter-action@v1.0.4
```

---

I didn't know GitHub Actions convention is to reassign the `vx` tags in repository to the latest revision for this major version. `v1` will be fine here for me.